index.bs

```diff
@@ -501,7 +491,7 @@ This specification defines a <a>policy-controlled feature</a> that controls
 whether the <a>request Picture-in-Picture algorithm</a> may return a
 {{SecurityError}} and whether {{pictureInPictureEnabled}} is `true` or `false`.
 
-The <a>feature name</a> for this feature is `"picture-in-picture"`.
+The feature name for this feature is `"picture-in-picture"`.
```

Comment:
If "feature name" doesn't exist anymore, shall we update spec text that says "If the document is not allowed to use the policy-controlled feature named "picture-in-picture", throw a SecurityError and abort these steps." as well?

Reply:
Actually, to avoid the dubious repetition "the feature name for this feature is", I just pushed an update to merge the sentence here with the previous paragraph .

Even though the term "feature name" is no longer defined, it still seems reasonable to talk about the name of a feature, so I didn't try to update the algorithm step. If you prefer, we could perhaps simply drop "named" in both cases to end up with 'policy-controlled feature "picture-in-picture"'.

Reply:
https://github.com/w3c/picture-in-picture/pull/174/commits/f4b49022fff416c0c9f71b332db6af1affdd3766 looks good to me.